src/modules/aws_account@0.0.1/index.ts

```diff
@@ -74,8 +73,4 @@ export const AwsAccount: Module = new Module({
       }),
     }),
   },
-  migrations: {
-    postinstall: awsAccount1644463885917.prototype.up,
-    preremove: awsAccount1644463885917.prototype.down,
-  },
-});
+}, __dirname);
```

Comment:
I would love to get rid of this bit here, but I couldn't figure out a way with the current structure.